scripts/docker_entrypoint_prod.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+poetry run gunicorn -b 0.0.0.0:$PORT 'src.app:create_app()'
```

Comment:
Was it worth creating this file for one command?

Reply:
I think I found that this didn't work properly when I had it in the dockerfile, I can't remember exactly what it was that didn't work though... Possibly using the environment variable for the port, not sure